app/src/main/res/values/strings.xml

```diff
@@ -2,7 +2,120 @@
     <string name="app_name">Cyrcle</string>
     <string name="default_web_client_id">836297757379-3rvkggdu23fdvnof5nko2lablc993e0u.apps.googleusercontent.com</string>
 
+    <!-- Common section -->
+    <string name="yes">Yes</string>
+    <string name="no">No</string>
+    <string name="no_selected_parking_error">No parking selected. Should not happen</string>
+    <string name="default_parking_name">Unnamed Parking</string>
+    <string name="distance_km">%.2f km</string>
+    <string name="distance_m">%.0f m</string>
+    <string name="no_reviews">No reviews yet</string>
+    <plurals name="reviews_count">
+        <item quantity="zero">(0 review)</item>
+        <item quantity="one">(1 review)</item>
```

Comment:
The `0` case will be assimilated with `others` , as it too require an *s*. So this issue shouldn't exist with the next commit.

As to why there is multiple cases, it is because I didn't find a way to declare multiple case (even thought there is, but I forgot to update this)